tests/js/client/http/coordinator-metrics-cluster.js

```diff
@@ -0,0 +1,129 @@
+/*jshint globalstrict:false, strict:false, maxlen: 500 */
+/*global fail, assertUndefined, assertEqual, assertNotEqual, assertTrue, assertFalse, assertNull, print*/
+
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2022 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Valery Mironov
+////////////////////////////////////////////////////////////////////////////////
+
+const jsunity = require("jsunity");
+const arangodb = require("@arangodb");
+const {getEndpointsByType, getMetricRaw} = require("@arangodb/test-helper");
+const parsePrometheusTextFormat = require("parse-prometheus-text-format");
+const _ = require("lodash");
+
+const db = arangodb.db;
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test suite
+////////////////////////////////////////////////////////////////////////////////
+
+function CoordinatorMetricsTestSuite() {
+    return {
+        setUpAll: function () {
+            db._useDatabase("_system");
```

Comment:
It shouldn't be necessary to explicitly move into the `_system` database here.
```suggestion
```

Reply:
Done